public/modules/custom/grants_webform_print/templates/grants-webform-print-webform.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="webform-print-page-body">
   <div class="container">
-    <p class="print-notification"><span aria-hidden="true" class="hel-icon hel-icon--info hds-icon--size-s"></span> {{ 'This printout is only for previewing the application and cannot be used when applying for a grant'|t }}</p>
+    <p class="print-notification"><img class="print-notification--icon" alt="Info icon" src="../../../../themes/custom/hdbt_subtheme/src/icons/hdbt/ui/info-circle.svg"> {{ 'This printout is only for previewing the application and cannot be used when applying for a grant'|t }}</p>
```

Comment:
the img tag could frankly have alt="" and aria-hidden="true" since the info icon isn't bringing any actual value to the person using a screen reader.